test/.eslintrc.js

```diff
@@ -1,3 +1,19 @@
+/*
```

Comment:
Hmm, is this desirable? other config/rc files don't usually have copyright but I'm not sure why not